app/models/generic_work.rb

```diff
@@ -11,6 +11,9 @@ class GenericWork < ActiveFedora::Base
   include SlugMetadata
   include AdventistMetadata
   include SlugBug
+  include IiifPrint.model_configuration(
+    pdf_split_child_model: GenericWork
```

Comment:
We may want this to eventually be a different model

Reply:
For now we thought we'd start with this, but we understand that it will likely change with the configuration ticket. However with this we will be able to demo the basics o Katharine.